runtime/mangata-kusama/src/lib.rs

```diff
@@ -1077,6 +1079,82 @@ impl xcm_asset_registry::Config for Runtime {
 	type TreasuryAddress = TreasuryAccount;
 }
 
+// Proxy Pallet
+/// The type used to represent the kinds of proxying allowed.
+#[derive(
+	Copy,
+	Clone,
+	Eq,
+	PartialEq,
+	Ord,
+	PartialOrd,
+	Encode,
+	Decode,
+	RuntimeDebug,
+	MaxEncodedLen,
+	TypeInfo,
+)]
+pub enum ProxyType {
+	Any,
+	CancelProxy,
+	AutoCompound,
+}
+
+impl Default for ProxyType {
+	fn default() -> Self {
+		Self::Any
+	}
+}
+
+parameter_types! {
+	pub const ProxyDepositBase: Balance = deposit(1, 16);
+	pub const ProxyDepositFactor: Balance = deposit(0, 33);
+	pub const AnnouncementDepositBase: Balance = deposit(1, 16);
+	pub const AnnouncementDepositFactor: Balance = deposit(0, 68);
+}
+
+impl InstanceFilter<Call> for ProxyType {
+	fn filter(&self, c: &Call) -> bool {
+		match self {
+			_ if matches!(c, Call::Utility(..)) => true,
+			ProxyType::Any => true,
+			ProxyType::CancelProxy =>
+				matches!(c, Call::Proxy(pallet_proxy::Call::reject_announcement { .. })),
+			ProxyType::AutoCompound => {
+				matches!(
+					c,
+					Call::Xyk(pallet_xyk::Call::mint_liquidity { .. }) |
+						// Call::Xyk(pallet_xyk::Call::claim_rewards_all { .. })
+						Call::Xyk(pallet_xyk::Call::claim_rewards { .. })
+				)
+			},
```

Comment:
to be defined later when we have a complete set of extrinsics working for auto compound
use this just as an example for now